src/corefx/System.Globalization.Native/calendarData.cpp

```diff
@@ -352,6 +460,7 @@ extern "C" int32_t EnumCalendarInfo(
 		case LongDates:
 			// TODO: need to replace the "EEEE"s with "dddd"s for .net
 			// Also, "LLLL"s to "MMMM"s
+			// Also, "G"s to "g"s
 			return InvokeCallbackForDateTimePattern(locale, "eeeeMMMMddyyyy", callback, context);
```

Comment:
This isn't a date pattern, but instead a date skeleton.  This string gets passed into [DateTimePatternGenerator::getBestPattern](http://userguide.icu-project.org/formatparse/datetime/examples), which returns the appropriate pattern for the locale, given the skeleton.  Basically this hard coded string says, give me a date pattern that contains the weekday, full month name, date of month and year.  Depending on the locale, it will return the best pattern for that locale.